plugins/hypervisors/vmware/test/com/cloud/hypervisor/vmware/resource/VmwareResourceTest.java

```diff
@@ -404,4 +410,46 @@ public void testConfigNestedHVSupportFlagFalse() throws Exception{
         verify(vmMo, never()).getRunningHost();
     }
 
+    @Test
+    public  void  testComposeVmNames() throws Exception{
```

Comment:
@priyankparihar please split the tests into multiple case methods, it's easier to debug failures.